### ISSUE TITLE
Add compatibility for PHP 8.2

### ISSUE THREAD

**Issue by lory-to** [firegento/firegento-magesetup2#215]
Magento2 v2.4.6 is now supporting PHP 8.2, this module fails to install due to missing compatibility in composer.json

### General

* Module version: 1.2.0
* Magento version: 2.4.6
* PHP version: 8.2.3

### Issue description

When upgrading PHP version to 8.2 and Magento to 2.4.6, after running `composer install/update` the following is displayed
```
firegento/magesetup2 dev-develop requires php ~7.3.0||~7.4.0||~8.1.0 -> your php version (8.2.3) does not satisfy that requirement.
```

### Steps to reproduce

1. Use PHP 8.2.3 Docker image
2. upgrade Magento package to 2.4.6 in composer.json
3. run `composer update`

**Comment by frostblogNet** [comment]
Hello @lory-to 

thanks you to mention. This is a open source project so if you want feel free to create a merge request.

**Comment by sprankhub** [comment]
Fixed in #216.
